LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
I would suspect many orgs do want that option the question is do we want to
encourage that? Xavier's point is that if orgs CAN close source their
changes, they will, which would mean less contributing back to the
community. He would like to see the opposite happen.

On Tue, Nov 24, 2015 at 8:56 AM, Joel Barciauskas <notifications@github.com>
wrote:

> In LICENSE
> <https://github.com/edx/xblock-lti-consumer/pull/1#discussion_r45737512>:
>
> > @@ -0,0 +1,202 @@
> > +
> > +                                 Apache License
>
> If that's the case, then I'm fine making this AGPL. I've mostly heard the
> opposite, though - most organizations want the option to close source their
> changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/xblock-lti-consumer/pull/1/files#r45737512>.
>